components/PicklistBase/PicklistBase.styling.comp.cy.js

```diff
@@ -156,7 +156,7 @@ describe('Test the PicklistBase component styling', () => {
     cy.get('@readySpy')
       .should('have.been.calledOnce')
       .then(() => {
-        cy.get('[data-cy="picklist-table"]').should('have.class', 'is-valid');
+        cy.get('[data-cy="picklist-table"]').should('have.class', '{}');
```

Comment:
This should check for `not.have.class` `is-valid` to be consistent with other tests.  In addition, this would fail if at sometime in the future someone added another style to the table for some other reason.